app/views/content_items/service_manual_guide.html.erb

```diff
@@ -78,5 +78,53 @@
       <%= render 'govuk_component/govspeak', content: @content_item.body.html_safe %>
     </div>
 
+    <div class="change-history" data-module="toggle">
+      <div class="change-history__latest-change">
+        <dl>
+        <% if @content_item.content_owners.any? %>
+          <dt>Published by:</dt>
+          <% @content_item.content_owners.each do |content_owner| %>
+            <dd>
+              <%= link_to content_owner.title, content_owner.href %>
+            </dd>
+          <% end %>
```

Comment:
Mentioned IRL but this could be partial'ed if it actually makes it more readable.